proxmoxtf/resource_virtual_environment_vm.go

```diff
@@ -326,6 +328,13 @@ func resourceVirtualEnvironmentVM() *schema.Resource {
 							ForceNew:     true,
 							ValidateFunc: getVMIDValidator(),
 						},
+						mkResourceVirtualEnvironmentVMCloneFull: {
+							Type:        schema.TypeBool,
+							Description: "The Clone Type, create a Full Clone (true) or a linked Clone (false)",
```

Comment:
Can you modify the description to something like the following to make it use the same phrasing as the other boolean arguments in the provider:

```
Whether to create a full clone (true) or a linked clone (false)
```